documentation/src/main/asciidoc/server_guide/server_guide.adoc

```diff
@@ -1007,3 +1046,38 @@ remoteCacheManager = new RemoteCacheManager(clientBuilder.build());
 
 Additionally, you might also want to enable client certificate authentication (and therefore also allow the use of the EXTERNAL SASL mech to authenticate clients).
 
+
+
+== Multi-tenancy
+
+Multi-tenancy allows accessing multiple containers as shown below:
+
+image::images/multi-tenancy.png.png[]
+
+Currently there are two supported protocols for accessing the data - using Hot Rod client and using REST interface.
+
+=== Using REST interface
+
+Multi-tenancy router uses URL prefixes to separate containers using the following scheme: `http://<server_ip>:<server_port>/rest/<rest_connector_name>/<cache_name>/<key>`. All HTTP operations remain exactly the same as using standard `rest-connector`.
```

Comment:
In the context of URLs, scheme means just the part before `://`, so I think it would better to call this a template.
Speaking of which, could the scheme in this template be `https` instead of `http`?